node/nodeTransactionsResults.go

```diff
@@ -77,6 +80,31 @@ func (n *Node) putSmartContractResultsInTransaction(
 			tx.SmartContractResults = append(tx.SmartContractResults, n.adaptSmartContractResult(scrHash, scr))
 		}
 	}
+
+	n.setStatusIfIsESDTTransferFail(tx)
+}
+
+func (n *Node) setStatusIfIsESDTTransferFail(tx *transaction.ApiTransactionResult) {
+	if len(tx.SmartContractResults) < 1 {
+		return
+	}
+
+	// check if is cross shard destination me
```

Comment:
isCrossShardTxDestMe instead of this comment

Reply:
done.